rules/falco_rules.yaml

```diff
@@ -1677,6 +1677,10 @@
 - list: trusted_images
   items: []
 
+# Backport compatibility
+- list: trusted_containers
```

Comment:
I think trusted_containers used to be a macro, right? In that case it should be a macro here and also it should be added to all the rules that might rely on that list.

Reply:
so it should be 
```
- macro: trusted_containers
  condition: container.image.repository in (trusted_images)
```